nodejs/UPDATING.md

```diff
@@ -4,16 +4,25 @@
 
 #### One-Time Setup
 
-1. Download and install [Node.js](https://www.npmjs.com/get-npm) (v10.15.2 LTS has been used)
+1. Download and install [Docker](https://docs.docker.com/get-docker/)
 
-#### Every time `gtfs-realtime.proto` changes
+#### Every time node package dependencies change
 
-1. Run `npm install` from the `nodejs` folder to install the protobuf package
+1. Download and install [Node.js](https://www.npmjs.com/get-npm) (check the `package.json` for `engines` version).
 
-1. Regenerate the language binding source from gtfs-realtime.proto by running the following from the `nodejs` folder:
+1. Edit the dependency versions in `package.json` and run `npm install`.
```

Comment:
The recommended way to modify dependencies is via `npm install` & `npm remove`; They will update both `package.json` & `package-lock.json` accordingly.